devlib/utils/gem5.py

```diff
@@ -39,5 +46,6 @@ def iter_statistics_dump(stats_file):
             if res:
                 k = res.group("key")
                 v = res.group("value").split()
+                v = map(best_effort_numeric, v)
```

Comment:
After an offline discussion with @Sticklyman1936, gem5 stats values will always be numeric (internally they're all represented as doubles), so best effort is not needed here.

We should assume that they are numeric, if, for some reason, the conversion fails, we do want that to be raised as an error (as that would indicate that something went wrong during stats collection) and not have it "swallowed" here.

However, we do want to make sure that, in such an event, the entire run should not be kills, so we do want to add a try/except guard to catch and log the error.

Not sure if it's best to do it here or further up the call stack?